app/assets/javascripts/modules/logger.js

```diff
@@ -0,0 +1,30 @@
+(function () {
+  "use strict";
+  window.GOVUK = window.GOVUK || {};
+
+  function Logger() {
+    if (isDevelopmentEnvironment()) {
+      return console;
+    } else {
+      return silentConsole();
+    }
+  }
+
+  function isDevelopmentEnvironment() {
+    return /\.dev\.gov\.uk$/.test(window.location.hostname);
+  }
+
+  function silentConsole() {
+    var fakeConsole = {};
+
+    $.each(console, function(key) {
+      if (typeof console[key] === 'function') {
+        fakeConsole[key] = function() {};
+      }
+    });
+
+    return fakeConsole;
+  }
+
+  GOVUK.Logger = Logger;
+})();
```

Comment:
it's not really abusing console imo, we have no release mechanism for static, so this is about the best you can do, i don't thunk there is a policy. Also can i ask why we are deprecating it? as in, it seems as if it could hang about?